devicetypes/axis/axis-gear-st.src/axis-gear-st.groovy

```diff
@@ -176,50 +177,47 @@ def pause() {
 //Send Command through setLevel()
 def on() {
 	log.info "on()"
-	sendEvent(name: "windowShade", value: "opening")
 	sendEvent(name: "switch", value: "on")
-	
-	if (state.currentVersion >= MIN_WINDOW_COVERING_VERSION) {
-		zigbee.command(CLUSTER_WINDOWCOVERING, WINDOWCOVERING_CMD_OPEN)
-	}
-	else {
-		setLevel(100)
-	}
+	open()
 }
 
 //Send Command through setLevel()
 def off() {
 	log.info "off()"
-	sendEvent(name: "windowShade", value: "closing")
 	sendEvent(name: "switch", value: "off")
 	close()
-	//zigbee.off()
 }
 
 //Command to set the blind position (%) and log the event
 def setLevel(value, rate=null) {
 	log.info "setLevel ($value)"
-	
+
+	setShadeLevel(value)
+}
+
+def setShadeLevel(value) {
+	log.info "setShadeLevel ($value)"
 	Integer currentLevel = state.level
-	
+
 	def i = value as Integer
-	sendEvent(name:"level", value: value, displayed:true)
-	
+	sendEvent(name:"level", value: value, unit:"%", displayed: false)
```

Comment:
is the unit necessary here? we don't have it anywhere else we use switch level afaik

Reply:
The unit is defaulted to `%`. I'm preferring to be pedantic with the use of units.